torch_xla/experimental/xla_sharded_tensor.py

```diff
@@ -58,7 +58,7 @@ class XLAShardedTensor(torch.Tensor):
   # 4-way row-wise, and replicate column-wise.
   # >> input = torch.randn(8, 10)
   # >> mesh_shape = (4, 2)
-  # >> assert np.prod(mesh_shape) == xm.xrt_world_size()
+  # >> assert np.prod(mesh_shape) == len(xm.get_xla_supported_devices())
```

Comment:
would it also include `xla:cpu`?

Reply:
Yes

Reply:
oh ok nvm. I am thinking about the case of XRT where we will have 1 CPU and 8 TPU. I think the PJRT case we will only have TPU as XLA device unless we specified `PJRT_DEVICE` to be CPU.